jolly-jellyfish/src/page_maker/templates/registration/login.html

```diff
@@ -1,24 +1,24 @@
 {% extends "page_maker/base.html" %}
 
 {% block content %}
-    {% if form.errors %}
-        <p>Your username and password didn't match. Please try again.</p>
-    {% endif %}
+<h2>Log in</h2>
+{% if form.errors %}
+<p>Your username and password didn't match. Please try again.</p>
+{% endif %}
 
-    <form method="post" action="{% url 'login' %}">
-    {% csrf_token %}
-         <table>
-         <tr>
-             <td>{{ form.username.label_tag }}</td>
-             <td>{{ form.username }}</td>
-         </tr>
-         <tr>
-             <td>{{ form.password.label_tag }}</td>
-             <td>{{ form.password }}</td>
-         </tr>
-         </table>
- 
-         <input type="submit" value="login" />
-         <input type="hidden" name="next" value="{{ next }}" />
-     </form>
+<form method="post" action="{% url 'login' %}">
+{% csrf_token %}
+  <table>
+  <tr>
+    <td>{{ form.username.label_tag }}</td>
+    <td>{{ form.username }}</td>
+  </tr>
+  <tr>
+    <td>{{ form.password.label_tag }}</td>
+    <td>{{ form.password }}</td>
+  </tr>
```

Comment:
```suggestion
    <tr>
      <td>{{ form.username.label_tag }}</td>
      <td>{{ form.username }}</td>
    </tr>
    <tr>
      <td>{{ form.password.label_tag }}</td>
      <td>{{ form.password }}</td>
    </tr>
```

very minor change though